tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/Shared/InterfaceTesting.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Globalization;
+
+namespace Samples.Probes.TestRuns.Shared
+{
+    public interface IGenericInterface<T>
+    {
+        public T Value { get; set; }
+    }
+
+    public interface IInterface
+    {
+#if NETCOREAPP3_1_OR_GREATER
+        // we should not see this property in the snapshot
+        public string CallToString => DateTime.Now.ToString(CultureInfo.CurrentCulture);
+#endif
+        public string DoNotShowMe { get; set; }
+        public string ShowMe { get; set; }
+    }
+
+    internal class Class : IInterface
+    {
+        internal string Field;
+
+        public Class()
+        {
+            Field = "I'm a class field";
+        }
+
+        private string _notAutoProp;
+        public string DoNotShowMe
+        {
+            get
+            {
+                _notAutoProp = ToString();
```

Comment:
```suggestion
                _notAutoProp = "This string should never be visible";
```

Reply:
Fixed in 963a0d8f0b02952b982c82fd16239363573c7b19